### YamlMime:ManagedReference
items:
- uid: System.Security.XmlSyntaxException
  id: XmlSyntaxException
  children:
  - System.Security.XmlSyntaxException.#ctor
  - System.Security.XmlSyntaxException.#ctor(System.Int32)
  - System.Security.XmlSyntaxException.#ctor(System.String)
  - System.Security.XmlSyntaxException.#ctor(System.Int32,System.String)
  - System.Security.XmlSyntaxException.#ctor(System.String,System.Exception)
  langs:
  - csharp
  name: XmlSyntaxException
  nameWithType: XmlSyntaxException
  fullName: System.Security.XmlSyntaxException
  type: Class
  summary: "Undantag som utlöses när det finns ett syntaxfel i XML-parsning. Den här klassen kan inte ärvas."
  remarks: "XmlSyntaxException använder förvalet <xref:System.Object.Equals%2A>-implementering som stöder referenslikheten.</xref:System.Object.Equals%2A>       En lista med inledande egenskapsvärden för en instans av XmlSyntaxException finns i konstruktorn."
  syntax:
    content: >-
      [System.Runtime.InteropServices.ComVisible(true)]

      public sealed class XmlSyntaxException : SystemException
  inheritance:
  - System.Exception
  - System.SystemException
  implements: []
  inheritedMembers: []
  platform:
  - net462
- uid: System.Security.XmlSyntaxException.#ctor
  id: '#ctor'
  parent: System.Security.XmlSyntaxException
  langs:
  - csharp
  name: XmlSyntaxException()
  nameWithType: XmlSyntaxException.XmlSyntaxException()
  fullName: System.Security.XmlSyntaxException.XmlSyntaxException()
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System.Security
  summary: "Initierar en ny instans av den <xref href=&quot;System.Security.XmlSyntaxException&quot;> </xref> klass med standardegenskaper."
  remarks: "I följande tabell visas de inledande egenskapsvärdena för en instans av <xref:System.Security.XmlSyntaxException>.</xref:System.Security.XmlSyntaxException>      | Egenskapen | Värdet |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| `null`.|   | <xref:System.Exception.Message%2A>|&quot; Ogiltig syntax. &quot;|</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public XmlSyntaxException ();
    parameters: []
  overload: System.Security.XmlSyntaxException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Security.XmlSyntaxException.#ctor(System.Int32)
  id: '#ctor(System.Int32)'
  parent: System.Security.XmlSyntaxException
  langs:
  - csharp
  name: XmlSyntaxException(Int32)
  nameWithType: XmlSyntaxException.XmlSyntaxException(Int32)
  fullName: System.Security.XmlSyntaxException.XmlSyntaxException(Int32)
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System.Security
  summary: "Initierar en ny instans av den <xref href=&quot;System.Security.XmlSyntaxException&quot;> </xref> klass med radnumret där undantag upptäcktes."
  remarks: "Radnumret för en XML-dataström uppdateras varje gång ett radmatningstecken har identifierats. Numreringen börjar med 1 för den första raden.       I följande tabell visas de inledande egenskapsvärdena för en instans av <xref:System.Security.XmlSyntaxException>.</xref:System.Security.XmlSyntaxException>      | Egenskapen | Värdet |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| `null`.|   | <xref:System.Exception.Message%2A>|&quot; Ogiltig syntax på rad `lineNumber`. &quot;|</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public XmlSyntaxException (int lineNumber);
    parameters:
    - id: lineNumber
      type: System.Int32
      description: "Radnumret för XML-dataström där XML-syntaxfel har identifierats."
  overload: System.Security.XmlSyntaxException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Security.XmlSyntaxException.#ctor(System.String)
  id: '#ctor(System.String)'
  parent: System.Security.XmlSyntaxException
  langs:
  - csharp
  name: XmlSyntaxException(String)
  nameWithType: XmlSyntaxException.XmlSyntaxException(String)
  fullName: System.Security.XmlSyntaxException.XmlSyntaxException(String)
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System.Security
  summary: "Initierar en ny instans av den <xref href=&quot;System.Security.XmlSyntaxException&quot;> </xref> klass med ett angivet meddelande."
  remarks: "I följande tabell visas de inledande egenskapsvärdena för en instans av <xref:System.Security.XmlSyntaxException>.</xref:System.Security.XmlSyntaxException>      | Egenskapen | Värdet |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| `null`.|   | <xref:System.Exception.Message%2A>| Meddelandet felsträng. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public XmlSyntaxException (string message);
    parameters:
    - id: message
      type: System.String
      description: "Felmeddelandet som förklarar orsaken till undantaget."
  overload: System.Security.XmlSyntaxException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Security.XmlSyntaxException.#ctor(System.Int32,System.String)
  id: '#ctor(System.Int32,System.String)'
  parent: System.Security.XmlSyntaxException
  langs:
  - csharp
  name: XmlSyntaxException(Int32,String)
  nameWithType: XmlSyntaxException.XmlSyntaxException(Int32,String)
  fullName: System.Security.XmlSyntaxException.XmlSyntaxException(Int32,String)
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System.Security
  summary: "Initierar en ny instans av den <xref href=&quot;System.Security.XmlSyntaxException&quot;> </xref> klass med ett angivet meddelande och radnumret där undantag upptäcktes."
  remarks: "Radnumret för en XML-dataström uppdateras varje gång ett radmatningstecken har identifierats. Numreringen börjar med 1 för den första raden.       I följande tabell visas de inledande egenskapsvärdena för en instans av <xref:System.Security.XmlSyntaxException>.</xref:System.Security.XmlSyntaxException>      | Egenskapen | Värdet |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| `null`.|   | <xref:System.Exception.Message%2A>|&quot; Ogiltig syntax på rad `lineNumber`  -  `message` &quot;|</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public XmlSyntaxException (int lineNumber, string message);
    parameters:
    - id: lineNumber
      type: System.Int32
      description: "Radnumret för XML-dataström där XML-syntaxfel har identifierats."
    - id: message
      type: System.String
      description: "Felmeddelandet som förklarar orsaken till undantaget."
  overload: System.Security.XmlSyntaxException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Security.XmlSyntaxException.#ctor(System.String,System.Exception)
  id: '#ctor(System.String,System.Exception)'
  parent: System.Security.XmlSyntaxException
  langs:
  - csharp
  name: XmlSyntaxException(String,Exception)
  nameWithType: XmlSyntaxException.XmlSyntaxException(String,Exception)
  fullName: System.Security.XmlSyntaxException.XmlSyntaxException(String,Exception)
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System.Security
  summary: "Initierar en ny instans av den <xref href=&quot;System.Security.XmlSyntaxException&quot;> </xref> klass med ett angivet meddelande och en referens till det interna undantaget som är orsaken till det här undantaget."
  remarks: "Ett undantag som utlöses när ett direkt resultat av ett tidigare undantag ska innehålla en referens till det tidigare undantaget i den <xref:System.Exception.InnerException%2A>egenskapen.</xref:System.Exception.InnerException%2A> Den <xref:System.Exception.InnerException%2A>-egenskap returnerar samma värde som överförs till konstruktorn eller `null` om den <xref:System.Exception.InnerException%2A>egenskapen tillhandahåller inte värdet i det interna undantaget för konstruktorn.</xref:System.Exception.InnerException%2A> </xref:System.Exception.InnerException%2A>       I följande tabell visas de inledande egenskapsvärdena för en instans av <xref:System.Security.XmlSyntaxException>.</xref:System.Security.XmlSyntaxException>      | Egenskapen | Värdet |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| Referens för inre undantag. |   | <xref:System.Exception.Message%2A>| Meddelandet felsträng. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public XmlSyntaxException (string message, Exception inner);
    parameters:
    - id: message
      type: System.String
      description: "Felmeddelandet som förklarar orsaken till undantaget."
    - id: inner
      type: System.Exception
      description: "Undantag som är orsaken till det aktuella undantaget. Om den `inner` parametern är inte <xref uid=&quot;langword_csharp_null&quot; name=&quot;null&quot; href=&quot;&quot;> </xref>, det aktuella undantaget utlöses i en <xref uid=&quot;langword_csharp_catch&quot; name=&quot;catch&quot; href=&quot;&quot;> </xref> block som hanterar det interna undantaget."
  overload: System.Security.XmlSyntaxException.#ctor*
  exceptions: []
  platform:
  - net462
references:
- uid: System.SystemException
  isExternal: false
  name: System.SystemException
- uid: System.Security.XmlSyntaxException.#ctor
  parent: System.Security.XmlSyntaxException
  isExternal: false
  name: XmlSyntaxException()
  nameWithType: XmlSyntaxException.XmlSyntaxException()
  fullName: System.Security.XmlSyntaxException.XmlSyntaxException()
- uid: System.Security.XmlSyntaxException.#ctor(System.Int32)
  parent: System.Security.XmlSyntaxException
  isExternal: false
  name: XmlSyntaxException(Int32)
  nameWithType: XmlSyntaxException.XmlSyntaxException(Int32)
  fullName: System.Security.XmlSyntaxException.XmlSyntaxException(Int32)
- uid: System.Int32
  parent: System
  isExternal: true
  name: Int32
  nameWithType: Int32
  fullName: System.Int32
- uid: System.Security.XmlSyntaxException.#ctor(System.String)
  parent: System.Security.XmlSyntaxException
  isExternal: false
  name: XmlSyntaxException(String)
  nameWithType: XmlSyntaxException.XmlSyntaxException(String)
  fullName: System.Security.XmlSyntaxException.XmlSyntaxException(String)
- uid: System.String
  parent: System
  isExternal: true
  name: String
  nameWithType: String
  fullName: System.String
- uid: System.Security.XmlSyntaxException.#ctor(System.Int32,System.String)
  parent: System.Security.XmlSyntaxException
  isExternal: false
  name: XmlSyntaxException(Int32,String)
  nameWithType: XmlSyntaxException.XmlSyntaxException(Int32,String)
  fullName: System.Security.XmlSyntaxException.XmlSyntaxException(Int32,String)
- uid: System.Security.XmlSyntaxException.#ctor(System.String,System.Exception)
  parent: System.Security.XmlSyntaxException
  isExternal: false
  name: XmlSyntaxException(String,Exception)
  nameWithType: XmlSyntaxException.XmlSyntaxException(String,Exception)
  fullName: System.Security.XmlSyntaxException.XmlSyntaxException(String,Exception)
- uid: System.Exception
  parent: System
  isExternal: true
  name: Exception
  nameWithType: Exception
  fullName: System.Exception
- uid: System.Security.XmlSyntaxException.#ctor*
  parent: System.Security.XmlSyntaxException
  isExternal: false
  name: XmlSyntaxException
  nameWithType: XmlSyntaxException.XmlSyntaxException
